numpy/lib/arraysetops.py

```diff
@@ -527,36 +527,36 @@ def isin(elements, test_elements, assume_unique=False, invert=False):
     Notes
     -----
 
-    `isin` can be considered as an element-wise function version of the
-    python keyword `in`. ``isin(a, b)`` is roughly equivalent to
+    `isin` is an element-wise function version of the python keyword `in`.
+    ``isin(a, b)`` is roughly equivalent to
     ``np.array([item in b for item in a])`` if `a` and `b` are 1-D sequences.
-    
-    If `test_elements` is a set (or other non-sequence collection) it will 
-    be converted to an object array with one element, rather than an array 
-    of the values contained in `test_elements`. Converting the set to 
+
+    If `test_elements` is a set (or other non-sequence collection) it will
+    be converted to an object array with one element, rather than an array
+    of the values contained in `test_elements`. Converting the set to
```

Comment:
Probably worth pointing out that this is just the `np.array` constructor doing its thing, and is not something weird this function chose to do. Mentioning "`array` constructor" somewhere should do the job